_includes/nav.html

```diff
@@ -32,6 +32,7 @@
             <a class="dropdown-item" href="/providers/search-engines/"><span class="fas fa-search fa-fw"></span> Search Engines</a>
             <a class="dropdown-item" href="/providers/social-networks/"><span class="fas fa-expand-arrows-alt fa-fw"></span> Social Networks</a>
             <a class="dropdown-item" href="/providers/social-news-aggregator/"><span class="far fa-newspaper fa-fw"></span> Social News Aggregators</a>
+            <a class="dropdown-item" href="/software/video-streaming/"><i class="fas fa-expand-arrows-alt fa-fw"></i> Video Streaming</a>            <a class="dropdown-item" href="/providers/vpn/"><span class="far fa-eye-slash fa-fw"></span> VPN</a>
```

Comment:
You've duplicated VPN in the navbar:

```suggestion
            <a class="dropdown-item" href="/software/video-streaming/"><i class="fas fa-expand-arrows-alt fa-fw"></i> Video Streaming</a>
```